packages/scanner/ml_kit/lib/src/scanner_ml_kit.dart

```diff
@@ -123,9 +154,9 @@ class _SmoothBarcodeScannerMLKitState extends State<SmoothBarcodeScannerMLKit>
                 child: SmoothBarcodeScannerVisor(),
               ),
             ),
-            const Align(
+            Align(
               alignment: Alignment.topCenter,
-              child: ScanHeader(),
+              child: widget.scanHeader,
```

Comment:
Suggestion: I wouldn't put the `ScanHeader` here.
In this class we just need a scanner (mlkit or zxing) and its specific buttons (flash, switch). The rest we can do on top of it (visor, top buttons).